epistles-v1/original/05-epistles.adoc

```diff
@@ -1189,4 +1190,5 @@ in the still life, patient, to the answering the good in all,
 to the refreshing one of another, and not to the stumbling.
 But mind that which keepeth in unity, in the life, though never so little.
 
+[.signed-section-signature]
```

Comment:
No problem with this line, but can you make lines 606-611 of this file (ch. 5) a _postscript_ while you’re at it? -- also line 981 - 983 ("And Friends, in the measure of life,...") -- basically any time you see him sign an epistle _twice_ the chunk after the first signature should be a PS. 👍